js/gratipay/edit_team.js

```diff
@@ -0,0 +1,51 @@
+Gratipay.edit_team = {}
+
+Gratipay.edit_team.initForm = function() {
+    $form = $("#edit-team");
+    $buttons = $form.find("button");
+
+    $form.find(":input").each(function() {
+        // Save original values.
+        $(this).data('original', $(this).val());
+    });
+    $form.submit(Gratipay.edit_team.submitForm);
+}
+
+Gratipay.edit_team.submitForm = function(e) {
+    e.preventDefault();
+
+    var modified = false;
+
+    var data = new FormData();
+    $form.find(":input").each(function() {
+        if($(this).data("original") !== $(this).val()) {
+            modified = true;
+
+            var name = $(this).attr("name");
+            if (name === "image") {
+                data.append(name, $(this)[0].files[0]);
+            } else {
+                data.append(name, $(this).val());
```

Comment:
> The idea here is that if later on we decide to expose this api to public officially, you shouldn't have to do a GET (to get all the attributes and their values) before hitting this endpoint.

I agree.

My question though, is why we're doing it in our internal usage. I'm concerned about this because a JS bug in the logic that figures out which fields have changed would be very hard to track down. Wouldn't this functionality work exactly the same if we weren't generating a diff (just send out all the values) before submitting the form here?